homeassistant/components/matter/cover.py

```diff
@@ -56,17 +57,42 @@ class MatterCover(MatterEntity, CoverEntity):
     """Representation of a Matter Cover."""
 
     entity_description: CoverEntityDescription
-    _attr_supported_features = (
-        CoverEntityFeature.OPEN
-        | CoverEntityFeature.CLOSE
-        | CoverEntityFeature.STOP
-        | CoverEntityFeature.SET_POSITION
-    )
 
     @property
-    def is_closed(self) -> bool:
-        """Return true if cover is closed, else False."""
-        return self.current_cover_position == 0
+    def is_closed(self) -> bool | None:
+        """Return true if cover is closed, if there is no position report, return None."""
+        has_current_position = self._entity_info.endpoint.has_attribute(
+            None, clusters.WindowCovering.Attributes.CurrentPositionLiftPercentage
+        )
+        has_current_tilt_position = self._entity_info.endpoint.has_attribute(
+            None, clusters.WindowCovering.Attributes.CurrentPositionTiltPercentage
+        )
+
+        match has_current_position, has_current_tilt_position:
+            case True, False:
+                return (
+                    self.current_cover_position == 0
+                    if self.current_cover_position is not None
+                    else None
+                )
+            case False, True:
+                return (
+                    self.current_cover_tilt_position == 0
```

Comment:
The `is_closed` attribute is only concerned with the main up down movement of a cover.

Reply:
are you sure ? what about a tilt only cover ?

Reply:
There's an architecture issue opened recently about this. We don't currently have a way to represent this.

Reply:
ah yes, true.

Reply:
Should I modify it? or can we leave until the issue is solved and then modify it?

Reply:
Best to remove it so we can merge the PR.

Reply:
omw